test-case/check-pause-resume.sh

```diff
@@ -99,15 +99,21 @@ do
     expect <<END
 spawn $cmd $SOF_ALSA_OPTS $cmd_opts -D $dev -r $rate -c $channel -f $fmt -vv -i $file_name -q
 set i 1
+set maxcount 1
 expect {
-    "*#*+*\%" {
+    -re "#.*\%\r" {
+        puts "\rFred: normal volume, maxcount reset to 1\n"
```

Comment:
Sorry, didn't delete debugging log, this and below
Hehe I deleted in multiple-pause-resume.sh, but not here.